src/plugins/visualizations/public/visualize_app/components/visualize_editor.tsx

```diff
@@ -33,6 +33,12 @@ export const VisualizeEditor = ({ onAppLeave }: VisualizeAppProps) => {
   const [eventEmitter] = useState(new EventEmitter());
   const [hasUnsavedChanges, setHasUnsavedChanges] = useState(!visualizationIdFromUrl);
 
+  useExecutionContext(services.executionContext, {
```

Comment:
@kibana-vis-editors is it possible to get the name of the editor in this file?

Reply:
I think you can inspect `visEditorController.vis.type.title` for that

Reply:
But what if this is a new, unsaved vis? Will it still be set?

Reply:
Worked!
I updated the PR and the screenshots